### DISCUSSION TITLE
Pouvoir acheter plusieurs potions de vie d'un coup

### DISCUSSION THREAD

**Discussion by Procion06** [Crownicles/Crownicles#1457] [Suggestions fermées]
Quand il y a des potions de vie dans le shop et qu'on veut remonter sa santé il faut parfois faire une trentaine de manips !
(ouvrir 10 fois le shop, acheter 10 potions, les boire)
Ce serait beaucoup plus simple pour les joueurs d'avoir une possibilité d'achat par "pack" comme ça se fait déjà pour la nourriture des pets
Je sais qu'on a des emplacements limités dans l'inventaire mais un emplacement dédié qui permettrait de stocker plusieurs popos de même catégorie (dans la limite du raisonnable et ne dépassant pas 17pv par popo par ex) serait un plus évident et souhaitable pour les joueurs !

**Comment by BastLast** [comment]
Woa la galère à dev

**Comment by BastLast** [comment]
Pr info le farm de petites popos de vie va etre nerf
